neuralcompression/layers/_hyper_analysis_transformation_2d.py

```diff
@@ -14,26 +16,52 @@ class HyperAnalysisTransformation2D(Module):
     signal from a latent representation.
 
     The method is described in:
+
         | “Variational Image Compression with a Scale Hyperprior”
         | Johannes Ballé, David Minnen, Saurabh Singh, Sung Jin Hwang,
             Nick Johnston
         | https://arxiv.org/abs/1802.01436
 
     Args:
-        in_channels: number of channels in the input signal.
-        out_channels: number of channels produced by the transformation.
+        network_channels: number of channels in the input signal.
+        compression_channels: number of channels produced by the transformation.
     """
 
-    def __init__(self, in_channels: int, out_channels: int):
+    def __init__(
+        self,
+        network_channels: int,
+        compression_channels: int,
+        in_channels: int = 3,
+        activation: Module = ReLU(inplace=True),
+    ):
         super(HyperAnalysisTransformation2D, self).__init__()
 
-        self.models = Sequential(
-            Conv2d(in_channels, out_channels, (3, 3), (1, 2), 2),
-            ReLU(inplace=True),
-            Conv2d(out_channels, out_channels, (5, 5), (2, 2), 2),
-            ReLU(inplace=True),
-            Conv2d(out_channels, out_channels, (5, 5), (2, 2), 2),
+        self.encode = Sequential(
+            AbsoluteValue(),
```

Comment:
I would use `torch.abs` in the `forward`.

Reply:
Done!